extensions/machine-learning/src/views/models/manageModels/manageModelsDialog.ts

```diff
@@ -41,6 +41,7 @@ export class ManageModelsDialog extends ModelViewBase {
 
 		let dialog = this.dialogView.createDialog(constants.viewImportModelsTitle, [this.currentLanguagesTab]);
 		dialog.isWide = true;
+		dialog.dialogStyle = 'flyout';
```

Comment:
we shouldn't need this, right? the default value should be the old behavior:flyout.

Reply:
@alanrenmsft I set default values in modal.ts; however, the Dialog code does not access this. I need to find a place where I can add default values. I tested the SQL Agent > + New Job dialog and found that it does not present as a flyout. It's unrelated to ML and so is a good example of something breaking with my work done in core.